### ISSUE TITLE
Warlock - Succubus seduce out of range

### ISSUE THREAD

**Issue by kysqt** [dalaranwow/dalaran-wow#5247]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The warlock pet Succubus when commanded to seduce focus or arena1/2/3/4/5 with macros will not run to the target to seduce, instead it will give an error message "Your pet is out of range" https://cdn.discordapp.com/attachments/308030845942366208/400417338337394688/WoWScrnShot_010918_233417.jpg  <<<screenshot

**Expected behaviour**: Succubus should run to the target you commanded it to cc.

**Steps to reproduce the problem**:

1. Make a warlock
2. Make this macro
#showtooltip
/clearfocus [modifier:alt]
/focus [target=focus,noexists]; [target=focus,dead]
/clearfocus [target=focus,help]
/petstay
/petfollow
/cast [pet:succubus,target=focus,exists,harm] Seduction; Seduction
3. Try to seduce a target more than 30 yards away

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by jegharikketid** [comment]
Hello, i really need your help for a macro like this but not for arena just for my current target. Please contact me!!!! its for vanilla wow btw

**Comment by kysqt** [comment]
I have no idea how macros work in Vanilla, maybe google can help you friend.
